skaffold/src/main/kotlin/com/google/container/tools/skaffold/SkaffoldFiles.kt

```diff
@@ -50,13 +51,20 @@ fun isSkaffoldFile(file: VirtualFile): Boolean {
     return false
 }
 
-/**
- * Finds all Skaffold configuration YAML files in the given project.
- *
- * @param project IDE project to search Skaffold file in
- * @return List of Skaffold configuration files in the project.
- */
-fun findSkaffoldFiles(project: Project): List<VirtualFile> {
-    return FileTypeIndex.getFiles(YAMLFileType.YML, GlobalSearchScope.allScope(project))
-        .filter { isSkaffoldFile(it) }
+/** Obtains current active implementation of [SkaffoldFileService] */
+fun getSkaffoldFileService() = ServiceManager.getService(SkaffoldFileService::class.java)!!
+
+/** IDE service for findind Skaffold files in the given IDE project. */
+class SkaffoldFileService {
```

Comment:
how did you decide which functions moved to here vs staying in `SkaffoldFiles.kt`? seems like there isn't a clear boundary now.

Reply:
You're right, both of them fit nicely in the new service class. Let me move the other one and rename the file as well.